dali/operators/math/expressions/arithmetic_meta.h

```diff
@@ -44,8 +45,29 @@ enum class ArithmeticOp : int {
   // Unary arithmetic ops
   plus,
   minus,
+  sqrt,
+  cbrt,
   exp,
   log,
+  log2,
+  log10,
+  abs,
+  fabs,
+  floor,
+  ceil,
```

Comment:
round?

Reply:
I will see how much time I can spare to delve into the rounding functions. Maybe this one should be a follow up.